### DISCUSSION TITLE
package publishing question...

### DISCUSSION THREAD

**Discussion by vpenades** [emgucv/emgutf#58] [Q&A]
I would like to ask which plans do you have to publish new nuget packages at nuget.org...

Emgu.CV has been update a month ago.... and I was thinking that Emgu.TF.* would come short after... but it doesn't seem the case...

**Comment by emgucv** [comment]
The most current Emgu TF release is 2.4.0. The most current Tensorflow release is 2.4.1. 

We typically have new release when Tensorflow minor version changes (e.g. the next one will be 2.5.0). We may create patch release (e.g. 2.4.1) but we will not be able to gurantee that or have any definite timeline.

**Reply by vpenades** [reply]
I understand, and I was expecting an answer along these lines. My only concern is about the nuget dependencies: Emgu.CV.runtime.windows now depends on Emgu.runtime.windows.msvc.rt.* for its internal dependencies, while the current Emgu.TF.Windows is still using its own binaries, which collision with the files of Emgu.runtime.windows.msvc.rt.

This is not a problem during development, but still an issue when publishing/deploying our solution.

**Reply by emgucv** [reply]
Working on a 2.4.1 release now.
